force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -176,7 +271,12 @@ public with sharing class VariablesDataTypesOperators {
      * @return The average of the three numbers.
      */
     public static Double averageOfThreeNumbers(Integer a, Integer b, Integer c) {
-        return null; // Replace null with the variable you used to store the result
+
+        final Integer avgThatguy = (a + b+ c) / 3; 
```

Comment:
You only want to make something final when you wont change it in the future. Usually this is just reserved for constants you declare at the top.